frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -66,7 +66,7 @@ export const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
   }, [cpu, dispatch, memory, nodesCount]);
 
   React.useEffect(() => {
-    const isFlexibleScaling = nodesCount && zonesCount < 3;
+    const isFlexibleScaling = !!(nodesCount && zonesCount < 3);
```

Comment:
shouldn't this be `!!(nodesCount>=3 && zonesCount < 3)` otherwise even one node is selected flexible scaling will get enabled.